.ci/macos/dependencies/gource.sh

```
@@ -0,0 +1,118 @@
+gource --output-custom-log repo.log my-repo-location/
+gource --output-custom-log repo.log my-repo-location/ --start-date "2019-01-01" --stop-date "2019-12-31"
+
+## path .ci/repo/ 
+gource --output-custom-log repo.log ../../ --start-date "2015-01-01" --stop-date "2025-01-18"
+gource --seconds-per-day 1 repo.log
+
+# ========================================================================================================
+# Customizing Gource
+# specific screen resolution
+gource -1280x1024 --camera-mode track --hide progress --output-ppm-stream ./out.ppm
+--background-colour 000000 --font-colour 336699 --highlight-users --highlight-colour ff0000
```

Shellcheck (0.10.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)